specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1788,6 +1962,68 @@
           "readOnly": true
         }
       }
+    },
+    "DedicatedSQLminimalTlsSettingsPatchInfo": {
```

Comment:
>DedicatedSQLminimalTlsSettingsPatchInfo [](http://example.com/codeflow?start=5&length=39)

PATCH should patch properties at the correct path.  This request body would have "minimalTlsVersion" as a top level property when its real path in the resource is "properties.minimalTlsVersion"

Reply:
@pilor : Pushed the latest changes to modify the payload. Please check